stable/influxdb/templates/backup-cronjob.yaml

```diff
@@ -62,4 +65,29 @@ spec:
                 value: {{ .Values.backup.destination}}
               - name: KEY_FILE
                 value: /var/secrets/google/key.json
+          {{- end }}
```

Comment:
This can be used. but, Is there have any other solution can make the backup function consistency, and other configs would be set in the value.yaml ?